jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -435,3 +444,27 @@ boolean databricksCodeChanged() {
     }
     return false
 }
+
+// Wait for the rapids plgin built tars to be ready
+def waitForPluginBuiltTar(String tar_path, String db_type='aws', int timeout=60) {
+    // get DB runtimes from Jenkinsfile: "DB_RUNTIME [space]+ values 'x.y', 'a.b'"
+    def dbJenkinsfile = 'jenkins/Jenkinsfile-blossom.premerge-databricks'
```

Comment:
I understand we need a single source of truth for DB runtimes list, 
can we try some other ways? this is too complicated to get just the list runtimes

e.g. provide get runtimes from get builder

Reply:
or try move the `stage('Databricks IT part2')` to be triggered by stage one

Reply:
discussed internally, let's go with the first one first. 

If we can figure out a better way to refactor our existing usages later,
then the second could be a follow-up later, thanks

Reply:
Moved the checking of `plugin built tar` into the Databricks Jenkinsfile